api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -252,7 +253,13 @@ private FirecloudWorkspace attemptFirecloudWorkspaceCreation(FirecloudWorkspaceI
 
     dbWorkspace.setBillingMigrationStatusEnum(BillingMigrationStatus.NEW);
 
-    updateWorkspaceBillingAccount(dbWorkspace, workspace.getBillingAccountName());
+    try {
+      workspaceService.updateWorkspaceBillingAccount(
```

Comment:
Yeah we should definitely have some cron cleanup for orphans here. Do you mind filing a ticket for later?

For example, if this request times out (but actually succeeds on the backend), we wind up with a defunct project attached to the user's billing account.